--- origsrc/libqinfinity-1.0b5/libqinfinity/CMakeLists.txt	2010-04-03 06:32:35.000000000 -0500
+++ src/libqinfinity-1.0b5/libqinfinity/CMakeLists.txt	2011-06-27 20:14:36.388908000 -0500
@@ -97,7 +97,9 @@ SET_TARGET_PROPERTIES(qinfinity
 TARGET_LINK_LIBRARIES(qinfinity ${LIBS})
 
 INSTALL(TARGETS qinfinity
-	DESTINATION lib${LIB_SUFFIX})
+	LIBRARY DESTINATION lib${LIB_SUFFIX}
+	RUNTIME DESTINATION bin
+	ARCHIVE DESTINATION lib${LIB_SUFFIX})
 
 INSTALL(FILES ${QInfinity_HDRS}
 	DESTINATION include/libqinfinity-1.0/libqinfinity)
